GameData/RealismOverhaul/RO_SuggestedMods/HabTech2/RO_HabTech2_modules.cfg

```diff
@@ -0,0 +1,522 @@
+
+//	Leonardo Permanent Multipurpose Module (PMM)
+//	Diameter: 4.57m (15ft)
+//	Length: 6.6m (22ft)
+//	Mass: 4,082kg (8,999lb)
+//	Source: http://spacecraftearth.com/#smoothscroll-portfolio-module
+
+@PART[ht2_MPLM]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@rescaleFactor = 1.81
+	@mass = 4.082
+	@title = ISS Leonardo PMM
+	@manufacturer = ASI
+	@description = The Leonardo Permanent Multipurpose Module (PMM) is a module of the International Space Station. It was flown into space aboard the Space Shuttle on STS-133 on 24 February 2011 and installed on 1 March. Leonardo is primarily used for storage of spares, supplies and waste on the ISS, which is currently stored in many different places within the space station. The Leonardo PMM was a Multi-Purpose Logistics Module (MPLM) before 2011, but was modified into its current configuration. It was formerly one of three MPLM used for bringing cargo to and from the ISS with the Space Shuttle. The module was named for Italian polymath Leonardo da Vinci.
+	@maxTemp = 1073.15
+
+	MODULE
+	{
+		name = ModuleFuelTanks
+		type = SeviceModule
+		basemass = -1
+		volume = 31000 // Change to 77k?
+		TANK
+		{
+			name = ElectricCharge
+			amount = 11000
+			maxAmount = 11000
+		}
+	}
+}
+
+@PART[ht2_handrail]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@rescaleFactor = 1.81
+	@title = ISS EVA/Service handrail
+	@manufacturer = NASA
+	@description = Handrails serve as mobility and stability aids for astronauts during extravehicular activity. Handrails are installed to define a safe path and provide sturdy anchors for the astronauts to use while they are working outside the spacecraft.
+	@mass = 0.001
+}
+
+//	Common Berthing Mechanism
+//	Diameter: 1.8m
+//	Length: 0.4m
+//	Mass: 200kg (440lb)
+//	Source: https://en.wikipedia.org/wiki/Common_Berthing_Mechanism
+@PART[ht2_CBM]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	// Multiple rescaleFactor exist on original part. Remove them first
+	!rescaleFactor,* = DEL
```

Comment:
That's broken.  Worth PR'ing back to the original mod?

Reply:
It's only a problem when patching it (so here). Not sure how likely it is to be included and a release made. We would need this pattern to avoid issues with older versions of the mod